src/micro_http/src/server.rs

```diff
@@ -429,7 +431,7 @@ impl HttpServer {
     /// }
     /// ```
     pub fn epoll_fd(&self) -> RawFd {
-        self.epoll_fd
+        self.epoll_fd.as_raw_fd()
```

Comment:
This is sort of unsafe to use like this because there is no guarantee that the fd is safe from the moment you called the function up to the point you're actually using it.
Since we do not hold a `RawFd` anymore, could we also remove the safety problem here by returning a reference to `epoll_fd`?

Reply:
:+1: Done.